### PR TITLE
Create sun features

### PR DESCRIPTION
**Pull request by Jean-PhilippeD**
Usefull for sun conditions such as is-it night or is-it day ?

### CODE REVIEW

File: api/core/sun/index.js [hunk 1]
```diff
@@ -1,2 +1,5 @@
 
-module.exports.init = require('./sun.init.js');
\ No newline at end of file
+module.exports.init = require('./sun.init.js');
+module.exports.state = require('./sun.state.js');
```
Comment (Pierre-Gilles) [inline api/core/sun/index.js]: Can we just rename the function "state" to "getState" ? :)

File: api/core/sun/sun.isItNight.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = isItNight;
+
+
+function isItNight(options){
```
Comment (Pierre-Gilles) [inline api/core/sun/sun.isItNight.js:4]: We should add tests to isItNight as well

File: api/core/sun/sun.isItDay.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = isItDay;
+function isItDay(options){
```
Comment (Pierre-Gilles) [inline api/core/sun/sun.isItDay.js:2]: We should add tests to isItDay

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Great PR thx ! I'm gonna merge it soon because I need it for the weather module, it's gonna add informations about the current sun status (day/night) 

Just modify the little suggestion I added, and it will be ok :) 

Sorry for the late answer

**Comment by Jean-PhilippeD** [comment]
I've made the changes you asked for.
You can check, it should be ok

**Comment by Pierre-Gilles** [comment]
Thx ! I'm merging this !

**Pull request merged**